### PR TITLE
Monk mob weapon damage adjustments to match PC

### PR DESCRIPTION
**Pull request by MowFord**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Recreated my PR as I was doing cleanup and didn't realize my older LSB PRs were linked to the Wings-XI repo

Please see https://github.com/LandSandBoat/server/pull/4674

TL;DR - monk mobs that hit twice due to having a h2h weapon should not have the same weapon damage as mobs that only hit once.

## Steps to test these changes

Monk mobs like Lycopodiums should have reduced damage, but monk mobs like Sea Monks in Arrapago Reef should be unchanged.

### CODE REVIEW

**Review by Xaver-DaRed — DISMISSED** [review]

**Comment by Xaver-DaRed** [comment]
Mobs do not actually have weapons. Their models may have them, but internally, they dont.

Wouldn't this change make it so all mobs acted as a h2h?

**Comment by Frankie-hz** [comment]
There is no evidence to show that mobs follow the same h2h formula as players as I said in the previous PR. The information I provided was given to me by Jimmayus and his work done for Eden damage formulas. As all other mob base damage we can see that is extremely unlikely that they follow anything player related as they do not have weapons.

Side note: Sea Monks are WAR/WAR

**Comment by MowFord** [comment]
Maybe there's evidence that retail calculates things differently, but I've yet to see any data.

I do know that lsb does assign a weapon type, skill type, and weapon damage. Currently monk mobs get the same weapon DMG as mobs that do not have h2h weapon skill. This causes them to deal essentially twice damage (slightly less due to str contribution to attack, since they have h2h skill)

**Comment by Frankie-hz** [comment]
I am still waiting for any retail data from you showing anything from retail. Show me more than a wiki link saying “players do it this way so mobs do too” when nothing that has already been proven about mob base damage follows players. 

I suggest going to a monk mob with known stats, calculating pdif estimates and providing actual data to say the damage is in line with what it is supposed to be.

**Comment by Frankie-hz** [comment]
And if you don’t have a retail account. I am sure someone in the capture discord will be happy to floor pdif on a monk mob for you to get the relevant data. https://discord.gg/4wmkEuj6

**Comment by MowFord** [comment]
> And if you don’t have a retail account. I am sure someone in the capture discord will be happy to floor pdif on a monk mob for you to get the relevant data. https://discord.gg/4wmkEuj6

No thanks

**Comment by TeoTwawki** [comment]
> No thanks

k then
